lib/hex_web/validation.ex

```diff
@@ -26,6 +26,24 @@ defmodule HexWeb.Validation do
     end)
   end
 
+  def validate_requirement(changeset, field) do
+    validate_change(changeset, field, fn key, req ->
+      cond do
+        is_nil(req) ->
+          # Temporary friendly error message until people update to hex 0.9.1
+          [{key, {"invalid requirement: #{inspect req}, use \">= 0.0.0\" instead", []}}]
```

Comment:
yes, validate_change won't be run if `:elixir` is not present in changes.

I noticed `""` is also allowed as it's converted to `nil`, I guess because of _empty values_ changeset feature. imho it's fine to leave it as is, but I can make it stricter